urdfenvs/urdfCommon/urdf_env.py

```diff
@@ -108,11 +107,39 @@ def check_box(
         return msg_ext
 
 
+def check_shape_dim(dim: np.ndarray, shape_type: str, dim_len: int, default: np.ndarray) -> np.ndarray:
```

Comment:
check_shape_dim suggests that it returns a boolean, but it actually returns a filtered shape dimension

Reply:
It has been renamed to filter_shape_dim( )